packages/react/src/feature-link-list/FeatureLinkListItem.tsx

```diff
@@ -0,0 +1,76 @@
+import { ReactNode } from 'react';
+import { LinkProps, packs } from '../core';
+import { TextLink, TextLinkExternal } from '../text-link';
+import { Box } from '../box';
+import { Flex } from '../flex';
+import { Stack } from '../stack';
+import { Text } from '../text';
+import { ArrowRightIcon } from '../icon';
+import { FeatureLinkListBackground, hoverColorMap } from './utils';
+
+export type FeatureLinkListItemProps = LinkProps & {
+	label: ReactNode;
+	description?: ReactNode;
+	background?: FeatureLinkListBackground;
+};
+
+export const FeatureLinkListItem = ({
+	label,
+	description,
+	background = 'body',
+	...props
+}: FeatureLinkListItemProps) => {
+	const TextComponent = props.target == '_blank' ? TextLinkExternal : TextLink;
+
+	return (
+		<Box
+			as="li"
+			borderBottom
+			paddingY={0.25}
```

Comment:
Still needs to be 0.5?

Reply:
According to the design, 0.5 is is correct. Here you can see the design overlaid on your implementation and how the spacing is out 
<img width="534" alt="image" src="https://github.com/user-attachments/assets/461e94a8-daea-41fc-904e-02f2250844a7">

Reply:
Compare that to 0.5 
<img width="620" alt="image" src="https://github.com/user-attachments/assets/222d0f7b-c5a0-4f5e-8675-96f545d12dba">

Reply:
Your latest change misaligns the arrow icon. If you're worried about the focus ring, then I think we canjust remove the padding on the root box, and change the inner Flex to `1.5`.

This renders the focus ring correctly offset against the whole item 
<img width="709" alt="image" src="https://github.com/user-attachments/assets/f4180535-6cdd-4a01-9bfa-0a568f5bfdf6">

Reply:
Seems rigtht to me 
<img width="660" alt="image" src="https://github.com/user-attachments/assets/9310aab8-a32d-4ddc-8128-2e73521c7a5f">